packages/fluentui/docs/src/prototypes/table/index.tsx

```diff
@@ -157,6 +169,23 @@ const rowsChannels = [
   },
 ];
 
+const columnsPerson = {
+  items: ['id', 'Name', 'Age', 'Picture'],
+};
+
+const rowsPerson = [
+  ['1', 'Roman van von der Longername', '30 years', 'None'],
+  ['2', 'Alex', '1 year', 'None'],
+  ['3', 'Ali', '30000000000000 years', 'None'],
+];
+
+const responsiveColumnsConfig = [
```

Comment:
Can we add also example where the priority is in ascending order, just for illustration that functionality too

Reply:
Done, I added one where we keep the first and last columns ( Imagining the scenario where the last column would have actions )